### ISSUE TITLE
Minor Clojure 1.11 Alpha 4 compatibility issue

### ISSUE THREAD

**Issue by seancorfield** [kkinnear/zprint#215]
Alpha 4 introduces a new `abs` function in `clojure.core` leading to these warnings:

```
WARNING: abs already refers to: #'clojure.core/abs in namespace: zprint.util, being replaced by: #'zprint.util/abs
WARNING: abs already refers to: #'clojure.core/abs in namespace: zprint.guide, being replaced by: #'zprint.util/abs
WARNING: abs already refers to: #'clojure.core/abs in namespace: zprint.range, being replaced by: #'zprint.range/abs
```

**Comment by kkinnear** [comment]
Thank you so much for reporting this!  I'll probably cut a new release with just a fix for this and some other small things fairly soon, instead of in a few months.

**Comment by kkinnear** [comment]
Fixed in `1.2.1`.  Thanks again for noticing and submitting this!
